manifests/init.pp

```diff
@@ -67,6 +70,7 @@
   Array $collect_exported_keys_tags         = [$::domain],
   Optional[String] $ssh_private_key_source  = undef,
   Optional[String] $ssh_public_key_source   = undef,
+  Integer[0, default] $logout_timeout       = 0,
```

Comment:
Might be better to have this be `Optional[Integer[0, default]]` and default to `undef`, more explanation on that further down.

Reply:
Can do